programs/ssl/ssl_base64_dump.c

```diff
@@ -185,6 +198,30 @@ void print_if_bit( const char *str, int bit, int val )
     }
 }
 
+const char * enable_disable( int val )
```

Comment:
Style: Put `*` next to the identifier, here and in all other places in all other commits

```diff
- const char * enable_disable( int val )
+ const char *enable_disable( int val )
```